source/agora/consensus/Genesis.d

```diff
@@ -111,14 +165,77 @@ version (unittest)
     // Check that the public key matches, temporarily
     unittest
     {
-        assert(getGenesisKeyPair().address == GenesisOutputAddress);
+        assert(getTestGenesisKeyPair().address == UnitTestGenesisOutputAddress);
     }
 }
 
+/// The genesis block as defined by CoinNet
+private immutable Block CoinNetGenesisBlock =
+{
+    header:
+    {
+        prev_block:  Hash.init,
+        height:      0,
+        merkle_root: CoinNetGenesisMerkleRoot,
+    },
```

Comment:
This comment may not be related to this PR.
Now we need to think about the validators who will make the next block from Genesis.

Reply:
That's why this PR was made, so the tests can define who the validators are so we can define various testing scenarios.

Or did you wonder about something else?

Reply:
There are no validators in the Genesis Block at this time.
Then there is no node to validator or nominator the next block.
I thought we should define a validator in Genesis Block.